src/scripts/main.js

```diff
@@ -1,7 +1,40 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const notification = document.createElement('div');
+
+  document.body.appendChild(notification);
+  notification.classList.add('notification');
+
```

Comment:
It might be beneficial to add a default case for the notification type to handle unexpected values. This ensures that the function behaves predictably even if an invalid type is passed.